components/definition/cellRules/NetPyNECellRule.js

```diff
@@ -77,19 +78,19 @@ export default class NetPyNECellRule extends React.Component {
             <b>Conditions:</b>
           </div>
 
-          <NetPyNEField id={"netParams.cellParams.conds.cellModel"} >
+          <NetPyNEField id={"netParams.cellParams.conds.cellType"} >
```

Comment:
Are you swapping these fields on purpose?

Reply:
cellType is a field that the user has to complete in order for netpyne to work, so I thought it would be better to have it first (I should open this in a different PR, but the change was minor so I decided to put it in here)

Reply:
OK, it is not a big deal but try to do this kind of changes in different branches. It was already a big PR